### ISSUE TITLE
not working in v8.0 beta ??

### ISSUE THREAD

**Issue by JonB256** [KulpLights/fpp-FPPMon#16]
Working on a Pi4 that was running 7.x Master but just today updated and now says v8.x - Master - 21
but on a Pi 3b+ with a new img install of v8.x - Master - 21, it will not show the Login / Password prompt, even after several reboots.

Works on:

8.x-master-21-g609c3ef2
Platform: | Raspberry Pi (Pi 4)
FPP OS Build: | v2023-07
OS Version: | Raspbian GNU/Linux 11 (bullseye)
Hardware Serial Number: | 10000000e9749a18
Kernel Version: | 6.1.21-v8+
System Boot Time: | 2024-01-24 16:27:08
fppd Uptime: | 03:24
Local Git Version: | 609c3ef ChangeLog
Remote Git Version: | 609c3ef

Not working on:

FPP Version: | 8.x-master-21-g609c3ef2
Platform: | Raspberry Pi (Pi 3 Model B+)
FPP OS Build: | v2024-01
OS Version: | Raspbian GNU/Linux 12 (bookworm)
Hardware Serial Number: | 00000000a5649596
Kernel Version: | 6.7.0-rc8-v7+
Local Git Version: | 609c3ef ChangeLog
Remote Git Version: | 609c3ef

**Comment by JonB256** [comment]
This is on the 3B+ that I normally ran FPP Mon on during Christmas (before this beta update)

2024-01-24 16:41:52.579 (2216) [Plugin] /opt/fpp/src/Plugins.cpp:483: Failed to find shlib /home/fpp/media/plugins/fpp-FPPMon/libfpp-FPPMon.so
2024-01-24 16:41:52.665 (2216) [Plugin] /opt/fpp/src/Plugins.cpp:483: Failed to find shlib /home/fpp/media/plugins/fpp-brightness/libfpp-brightness.so

**Comment by brianbarrell** [comment]
Exact same issue here.  When you load the FPPmon page, you can briefly see the username and password fields and then it disappears.  I have tried on a Pi3B+ and on a K16A-B with a BBB.  Same issues on both.  Very frustrating since it worked great on previous version.

It simply displays this message  "FPP Remote Monitoring Plugin Not Running. Restart FPPD to enable. "

No restart will fix issue.  I have tried uninstalling and reinstalling plugin with same results.

**Comment by AGFazio** [comment]
Same issue here.

**Comment by JerryPlak** [comment]
adding me to list "FPP Remote Monitoring Plugin Not Running. Restart FPPD to enable. "

**Comment by dartunder** [comment]
Yes, it is not working on a fresh install of beta 2 and still didn't work after an FPP update. (to the latest patch 2024-08-23 22:41:25)
It doesn't work after restart or reboot; re-installing doesn't help.

FPP Version: | 8.x-master-471-gd3a9b9c0
Platform: | Raspberry Pi (Pi 5)
FPP OS Build: | v2024-08
OS Version: | Raspbian GNU/Linux 12 (bookworm)
Kernel Version: | 6.6.45-v8+
Local Git Version: | d3a9b9c
